pkg/trace/info/info.go

```diff
@@ -257,11 +258,16 @@ func InitInfo(conf *config.AgentConfig) error {
 			return
 		}
 
+		scrubbed, err := scrubber.ScrubBytes(buf)
```

Comment:
Can we add a test to make sure the info that we expect gets scrubbed correctly?

Reply:
[Added](https://github.com/DataDog/datadog-agent/pull/15591/commits/941ab27b9e5d2003b77bcfa0b13f28cda090f08b)!